crates/tessera-backbone/src/server/router/path/mod.rs

```diff
@@ -59,6 +62,22 @@ async fn handle_delete_path(
     Ok(StatusCode::NO_CONTENT)
 }
 
+#[debug_handler]
+async fn handle_patch_path(
+    Path((workspace_name, path)): Path<(String, String)>,
+    State(application): State<Arc<Application>>,
+    Json(payload): Json<PatchPathRequest>,
+) -> Result<impl IntoResponse, application::path::Error> {
+    if path == "/" || path.is_empty() {
```

Comment:
The current path handling logic needs to be reviewed for edge cases. While checking for `/` and empty paths, we should also consider cases where **paths end with a trailing slash**.  

According to the axum documentation (https://docs.rs/axum/latest/axum/struct.Router.html#wildcards), empty paths are not explicitly covered in the routing behavior. Additionally, the current implementation might need to normalize paths with trailing slashes to ensure consistent routing.  

Could you please verify the behavior with these edge cases and update the handling accordingly?